src/main/java/com/rcv/GuiConfigController.java

```diff
@@ -177,7 +177,7 @@
 
   public void buttonNewConfigClicked() {
     if (checkForSaveAndContinue()) {
-      Logger.log(Level.INFO, "Creating new contest config...");
+      Logger.log(Level.INFO, "Creating new contest config.");
```

Comment:
. implies action that completed, so keep as ... Please apply to / revert other cases.

Reply:
So I put this back since you like it, but I respectfully disagree with your assessment.  The . implies only the end of a sentence.  In this case the word "Creating" says explicitly, I am currently, or about to do some creating.  Is there some style guide you are drawing from?

Reply:
Appreciate it. I spent a good amount of time a while back checking all the log messages and making these changes to bring them in line with standard CLI messaging. I think the logic behind this is because it helps the user tell when something is frozen / still processing (e.g. if the last line was a ...) vs. safe to stop / kill (e.g. if the last line was a .). It also helps a lot with bug reports. So the idea was to creates pairs of "Doing this..." "This complete." statements so there's a clear delineation between what's happening and what's already finished. Hopefully that makes sense.

Reply:
Sure I get the concept.  What is this 'standard CLI messaging' you're talking about?